samples/expert-irb-adjusted.rb

```diff
@@ -44,10 +44,7 @@ def run(str)
     $stdout.rewind
     [output, obj]
   rescue Object => e
-    case e
-    when Empty, Continue
-    else @line = ""
-    end
+    @line = "" unless e == Empty || e == Continue
```

Comment:
Rubocop was complaining about the empty `when`, but I made the change here - we're through all the auto-correctable cops now. I wasn't sure about this change though, so I mentioned it in the PR comments.  Thanks for catching that I should have used `===`. How do you want this to look though, would this be better?

`@line = "" unless [Empty, Continue].include?(e.class)`